Assets/HoloToolkit/Utilities/Scripts/FixedAngularSize.cs

```diff
@@ -38,10 +49,25 @@ void Start()
             }
         }
 
+        public void SetSizeRatio(float ratio)
```

Comment:
This is meant to be called outside the class. I have a demo that uses it, but it may be a very rare case to need to update this on the fly. I should add a tooltip to it either way.